pkg/dockerutil/dockerutils.go

```diff
@@ -839,6 +849,14 @@ func CheckDockerVersion(versionConstraint string) error {
 		}
 		return fmt.Errorf("%s", msgs)
 	}
+	
+	dockerAPIVersion, err := GetDockerAPIVersion()
+	if err != nil {
+		return fmt.Errorf("unable to determine Docker API version: %v", err)
+	}
+	if !dockerVersions.GreaterThanOrEqualTo(dockerAPIVersion, minimalDockerAPIVersion) {
+		return fmt.Errorf("your version of Docker API (%s) doesn't meet the minimum (%s) - see (%s) for more information", dockerAPIVersion, minimalDockerAPIVersion, "https://ddev.rtfd.io/page/users/install/docker-installation/#testing-and-troubleshooting-your-docker-installation")
```

Comment:
I think we can simply add a link to Docker API matrix table https://docs.docker.com/reference/api/engine/#api-version-matrix

Because people won't understand what version is required if they don't see that table.